server/athenian/api/controllers/user_controller.py

```diff
@@ -32,3 +32,23 @@
     users = await request.auth.get_users(regulars + admins)
     account = Account(regulars=users[:len(regulars)], admins=users[len(regulars):])
     return response(account)
+
+
+async def become_user(request: AthenianWebRequest, id: str) -> web.Response:
+    """God mode ability to turn into any user. The current user must be marked internally as \
+    a super admin."""
+    user_id = getattr(request.user, "god_id", None)
+    if user_id is None:
+        return ResponseError(ForbiddenError(
+            detail="User %s is not allowed to mutate" % user_id)).response
+    async with request.sdb.connection() as conn:
+        god = await conn.fetch_one(select([God]).where(God.user_id == user_id))
+        if id and (await conn.fetch_one(
```

Comment:
We can swap these two lines to avoid the query if the user doesn't exist.

Reply:
Yes, we can. But I don't want to expose an endpoint that will allow everybody to count how many users we have registered by applying binary search.

Reply:
:+1: